frontend/src/components/Common/DateRangeViewer/style.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled'
+import { ArrowForward, CalendarToday } from '@mui/icons-material'
+
+export const DateRangeContainer = styled.div({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  backgroundColor: '#f3f3f3',
```

Comment:
Can we use the defined value of `theme` to replace it. or and it into `theme`

Try to reduce the new color value

Reply:
Of course yes. just requires a bit of UX support.